src/iamserver/nodecontroller.cpp

```diff
@@ -9,108 +9,7 @@
 
 #include "log.hpp"
 #include "nodecontroller.hpp"
-
-/***********************************************************************************************************************
- * Static
- **********************************************************************************************************************/
-
-static aos::Error ConvertToAos(
```

Comment:
It should be done as separate commit as add proto conversion to utils. And then use it in nodecontroller.